packages/fxa-react/configs/tailwind.js

```diff
@@ -59,6 +59,10 @@ module.exports = {
         '-3': '-0.75rem',
         '-52': '-13rem',
         '55': '13.75rem',
+        ten: '10%',
+        full: '100%',
+        center: '44%',
+        beyond: '140%',
```

Comment:
This also seems to cause issues with the tooltip positioning because it's a specific relative number. Check out where this bottom-anchored tooltip ends up when the content grows:

<img width="338" alt="Screen Shot 2020-09-22 at 11 37 20 AM" src="https://user-images.githubusercontent.com/6392049/93905108-9748a280-fcc8-11ea-9243-cbb7a15b7398.png">

Reply:
I guess we can just use 100% and let the developer add margin classes passed into the Tooltip component instead